src/main/resources/avro/variants.avdl

```diff
@@ -61,6 +94,15 @@ record VariantSet {
 }
 
 /**
+Some CallSets reflect genotype calls, while others reflect linked haplotype
+calls.
+*/
+enum CallSetType {
+  GENOTYPE,
+  HAPLOTYPE
+}
+
+/**
 A `CallSet` is a collection of variant calls for a particular sample.
 It belongs to a `VariantSet`. This is equivalent to one column in VCF.
```

Comment:
Does a `CallSet` belong to one `VariantSet` or many? The comment here implies one, but the `variantSetIds` list below implies many. I realise this is inherited from the main branch and we may not want to change too much to make transitioning between APIs as simple as possible. It would be good to clarify this relationship as far as the Graph based model is concerned here in a comment though.